examples/01-wsi-reading.ipynb

```diff
@@ -73,7 +73,7 @@
    "outputs": [],
```

Comment:
<p>Why it is replaced with @HEAD?</p><p>This doesn't seem right.</p><p>Also there are more changes in the rest of the notebooks. Any reasons why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TissueImageAnalytics/tiatoolbox/pull/519/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I have tried different options but it didn't work.
Can we not pass it as an argument?

Reply:
<p>HEAD is the current ref (e.g. the branch name) see, <a href="https://initialcommit.com/blog/what-is-git-head#git-refs-and-heads" target="_blank">https://initialcommit.com/blog/what-is-git-head#git-refs-and-heads</a></p>

Reply:
<p>Pass what as an argument?</p>

Reply:
Pass branch name as an argument, if possible.

Reply:
> HEAD is the current ref (e.g. the branch name) see, https://initialcommit.com/blog/what-is-git-head#git-refs-and-heads. It appears that the command to get the current ref is returning the shorthand HEAD instead of the branch name. I am not able to replicate this. Can you provide steps?

This happens online only. Locally it runs fine.